scripts/eurostat/regional_statistics_by_nuts/birth_death_migration/import_data.py

```diff
@@ -13,17 +13,44 @@
 # limitations under the License.
 
 import pandas as pd
+from six.moves import urllib
+import sys
 
+sys.path.insert(1, '../../../../util')
+from alpha2_to_dcid import COUNTRY_MAP
+from nuts_codes_names import NUTS1_CODES_NAMES
+from absl import app
+from absl import logging
+from absl import flags
+import os
 
-def download_data(download_link):
+_FLAGS = flags.FLAGS
+_MODULE_DIR = os.path.dirname(os.path.abspath(__file__))
+flags.DEFINE_string('mode', '', 'Options: download or process')
+
+
+def download_data(download_link, download_path):
     """Downloads raw data from Eurostat website and stores it in instance
-    data frame.
+       data frame.
+    
+        Args:
+        download_link(str): A string representing the URL of the data source.
+        download_path(str): A string specifying the local file path where the downloaded data will be saved.
+        
+        Returns:None
+        
     """
-    raw_df = pd.read_table(download_link)
-    return raw_df
+    logging.info("file downloading")
```

Comment:
can add the download link too:
`logging.info(f'Downloading: {download_link}')`

Reply:
Download link added